powerspy.py

```diff
@@ -127,6 +128,8 @@ def recvCmd(self, size = 1):
         buf = mat.group(1)
         logging.debug("RECV: <%s>" % buf)
         break
+    buf = buf.replace("'b'", "") # Fix receiving 'b' on some powerspy connections
```

Comment:
I think the fix is
```
      buf = "%s%s" % (buf,r.decode())
```
instead of  `buf = "%s%s" % (buf,r)`.
The reason is that `recv` will return `bytes` and not `str`.

Reply:
Yes, indeed. Works great. I'll push it.